### ISSUE TITLE
zypper: No provider of 'luarocks' found.

### ISSUE THREAD

**Issue by jayvdb** [coala/docker-coala-base#211]
All new builds are failing with

```
'luarocks' not found in package names. Trying capabilities.

No provider of 'luarocks' found.
```
https://travis-ci.org/coala/docker-coala-base/builds/253613612

**Comment by jayvdb** [comment]
I believe this is caused by the openSUSE Factory repository removing the generic 'luarocks'
package.
Instead they have `lua5x-luarocks`.
Switching to these version specific packages is useful for us anyway, as it ensures we have a more stable set of versions exposed to our users, where we explicitly change versions (and document them) instead of surprise upgrades whenever Tumbleweed (or Leap) changes its default.

The release 0.11 included lua 5.3.3 (see https://github.com/coala/docker-coala-base/issues/144#issuecomment-315105919)

**Comment by jayvdb** [comment]
c.f. https://gitlab.com/coala/package_manager/issues/156 this is also happening on Leap.
